test/extended/util/annotate/rules.go

```diff
@@ -329,6 +329,18 @@ var (
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1825027
 			`\[Feature:Platform\] Managed cluster should ensure control plane operators do not make themselves unevictable`,
 		},
+
+		// TODO: to facilitate v.14 rebase, skip the following tests until May 27 2022,
+		//  the following key should be removed after the rebase PR lands
+		//  BZs to keep track of these issues:
+		//   - [sig-api-machinery] API data in etcd should be: https://bugzilla.redhat.com/show_bug.cgi?id=2081021
+		//   - [sig-instrumentation] Events API should ensure that: https://bugzilla.redhat.com/show_bug.cgi?id=2081084
+		//   - [sig-auth] ServiceAccounts : https//bugzilla.redhat.com/show_bug.cgi?id=2081087
+		"[SkippedUntil:05272022]": {
```

Comment:
since we check it in `isDisabled` we could rename to `[DisabledUntil...]`

Reply:
so we use both `Disabled` and `Skipped` - i don't know what the semantic differences are between these two. Also i want to avoid a `DisabledUntil:` tag get misinterpreted as plain `Disabled`
```
func isDisabled(name string) bool {
	return strings.Contains(name, "[Disabled") 
}
 ```